### DISCUSSION TITLE
Every single package showing as needing to be updated has already been udpated

### DISCUSSION THREAD

**Discussion by aagha** [Devolutions/UniGetUI#1038] [General]
This is happening with every single package:

![image](https://github.com/marticliment/WingetUI/assets/5522103/1ffc5e41-86f0-4ab8-9d0e-cf04e401cb13)

Is there a way to force WingetUI to make sure it has the correct Installed Version?

**Comment by marticliment** [comment]
Hello,

does the command `winget update` list all of those already up-to-date packages?
